esp/src/src-react/components/LandingZone.tsx

```diff
@@ -358,6 +475,56 @@ export const LandingZone: React.FunctionComponent<LandingZoneProps> = ({
         }
     }, [setShowFileUpload, setUploadFiles]);
 
+    const onOpenChange = React.useCallback((evt: TreeOpenChangeEvent, data: TreeOpenChangeData) => {
+        const branchData = JSON.parse(data?.target?.dataset?.tree ?? "") ?? {};
+        if (data.type === "Click" || data.type === "Enter") {
+            const path = branchData.path[0] === "/" ? branchData.path : selectedDropzone.Path + "/" + branchData.path;
+            if (path !== lzPath) {
+                pushUrl(`#/landingzone/${path.replace(/\//g, "::")}`);
```

Comment:
See previous comment re capitalisation.  Might be worth opening a new ticket to address separately